cpp/src/arrow/parquet/parquet-io-test.cc

```diff
@@ -196,13 +290,31 @@ TEST_F(TestParquetIO, SingleColumnInt64ChunkedWrite) {
   ASSERT_TRUE(values->Equals(out));
 }
 
+TEST_F(TestParquetIO, SingleColumnTableInt64ChunkedWrite) {
```

Comment:
Will do, will wait for a review of https://github.com/apache/parquet-cpp/pull/113 as this is basically the same just a level deeper in the dependency chain.